appside/intro.js

```diff
@@ -0,0 +1,36 @@
+let name = 'Prototype 1';
```

Comment:
Eventually this data will be an object passed in to the intro rendering function as a parameter